omnibus/config/projects/agent.rb

```diff
@@ -2,7 +2,6 @@
 # under the Apache License Version 2.0.
```

Comment:
Most of the diff is because I've re-arranged the dependencies sequence. Actual changes are commented.